### PR TITLE
[SPARK-13874][Doc]Remove docs of streaming-akka, streaming-zeromq, streaming-mqtt and streaming-twitter

### PR DESCRIPTION
**Pull request by zsxwing**
## What changes were proposed in this pull request?

This PR removes all docs about the old streaming-akka, streaming-zeromq, streaming-mqtt and streaming-twitter projects since I have already copied them to https://github.com/spark-packages

Also remove mqtt_wordcount.py that I forgot to remove previously.
## How was this patch tested?

Jenkins PR Build.

### CODE REVIEW

File: docs/streaming-programming-guide.md [hunk 1]
```diff
@@ -2443,29 +2401,26 @@ Please refer to the project for more details.
 # Where to Go from Here
 * Additional guides
     - [Kafka Integration Guide](streaming-kafka-integration.html)
-    - [Flume Integration Guide](streaming-flume-integration.html)
     - [Kinesis Integration Guide](streaming-kinesis-integration.html)
     - [Custom Receiver Guide](streaming-custom-receivers.html)
+* External DStream data sources:
+    - [DStream Flume](https://github.com/spark-packages/dstream-flume)
```
Comment (zsxwing) [inline docs/streaming-programming-guide.md]: I also added their new locations here.

**Comment by zsxwing** [comment]
cc @rxin

**Comment by srowen** [comment]
This will need to add back in the Flume docs, but otherwise OK.

We should also remove references in `LICENSE`, `NOTICE`, and `licenses/`. It looks like the only instance, however, that is relevant, is this in `NOTICE`:

```
     (Eclipse Public License - Version 1.0) mqtt-client (org.eclipse.paho:mqtt-client:0.4.0 - http://www.eclipse.org/paho/mqtt-client)
```

Bonus: while you're editing those, if you like, you can remove `licenses/LICENSE-jblas.txt`. I removed the last usages of it in `master` and looks like I forgot to remove this.

I wondered if we also needed to update our dependency files that enumerate the transitivie dependencies for various builds, but doesn't look like it.

`python/docs/pyspark.streaming.rst` still has a reference to MQTT docs.

Finally: at the very end of `MimaBuild.scala` you can remove the MQTT resolver, I think.

**Comment by zsxwing** [comment]
Retest this please

**Comment by srowen** [comment]
LGTM pending tests. Let's try again

**Comment by srowen** [comment]
Jenkins retest this please

**Comment by zsxwing** [comment]
retest this please

**Comment by zsxwing** [comment]
retest this please

**Comment by zsxwing** [comment]
retest this please

**Comment by zsxwing** [comment]
retest this please

**Comment by zsxwing** [comment]
retest this please

**Comment by rxin** [comment]
Thanks - merging in master.

**Pull request closed without merging**